xml/signals/DB-HV-1969/appearance-block.xml

```diff
@@ -64,7 +64,7 @@
 
   <aspecttable>DB-HV-1969</aspecttable>
   <name>Single block signal</name>
-  <reference>See <a href="http://www.sh1.org/eisenbahn/shhp.htm">sh1's page</a> or the <a href="http://en.wikipedia.org/wiki/German_railway_signalling#Home_signal.2Fdistant_signal_.28H.2FV.29">Wikipedia</a></reference>
+  <reference>See &lt;a href="http://www.sh1.org/eisenbahn/shhp.htm"&gt;sh1's page&lt;/a&gt; or the &lt;a href="http://en.wikipedia.org/wiki/German_railway_signalling#Home_signal.2Fdistant_signal_.28H.2FV.29"&gt;Wikipedia&lt;/a&gt;</reference>
```

Comment:
Will this still display properly once it's run through the XSLT transform on the web server?  It's meant to be a link.  C.f. https://www.jmri.org/xml/signals/DB-HV-1969/appearance-block.xml

Reply:
No, it doesn't work. I created PR [#492](https://github.com/JMRI/website/pull/492) to test and this file failed:
https://www.jmri.org/download/danielb987/DB-HV-1969/appearance-block.xml

I'm trying to make the code work without this change to the xml files.

Reply:
It turned out to be a Catch-22 problem to solve this.

It's easy to read the reference and its child elements, the html links, and then write it out to the new xml document. BUT, the space before the link is removed unless I write the xml file with `Format.TextMode.PRESERVE`. But if I write the file with PRESERVE, the entire xml document will be on a single row. All the new line characters will be removed. So it's a Catch-22.

It seemed to me that the problem is HTML in a XML document. So my solution was to add optional elements `<referenceWithLinks>` and `<descriptionWithLinks>` to the schema file and then update the xml files accordingly.

But I don't know how to update the XSLT file.